docs/guidance/pod-security.md

```diff
@@ -0,0 +1,96 @@
+# Pod Security
+
+Kubernetes defines three different Pod Security Standards, including `privileged`, `baseline`, and `restricted`, to broadly
+cover the security spectrum. The `privileged` standard allows users to do known privilege escalations, and thus it is not 
+safe enough for security-critical applications.
+
+This document describes how to configure RayCluster YAML file to apply `restricted` Pod security standard. The following 
+references can help you understand this document better:
+
+* [Kubernetes - Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted)
+* [Kubernetes - Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/)
+* [Kubernetes - Auditing](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/)
+* [KinD - Auditing](https://kind.sigs.k8s.io/docs/user/auditing/)
+
+# Step 1: Create a KinD cluster
+```bash
+# Path: ray-operator/config/security
+kind create cluster --config kind-config.yaml --image=kindest/node:v1.24.0
```

Comment:
I ran out of memory on my (intel) MacBook -- the worker pod couldn't get scheduled.
Could we add an arg or config parameter that guarantees enough memory?
We should probably also mention that this doesn't work on ARM machines, like M1 books.

Reply:
Or mention the memory requirement.
Looks like I need to adjust my Docker setup.

Reply:
Good catch!

Reply:
Or reduce the memory requests -- do we really need 4GB per Ray pod?

Reply:
I will try it.

Reply:
Reduce memory requests.